### PR TITLE
Arkas main and 4 configs

### PR DESCRIPTION
**Pull request by linuxgurugamer**

### CODE REVIEW

File: NetKAN/Arkas.netkan [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	"resources" :
+	{
+		"homepage"	:	"http://forum.kerbalspaceprogram.com/threads/134547",
+		"kerbalstuff"	:	"https://kerbalstuff.com/mod/1158/Arkas%20-%20A%20Thick%20Atmosphere%20on%20a%20Desert%20World"
+	},
+	"depends"	:	[ { "name"	:	"Kopernicus", "min version": "0.4" } ],
+	"recommends"	:	[ { "name"	:	"Arkas-Config" } ],
+	"install"	:	[
+		{
+			"find"		:	"Arkas",
```
Comment (plague006) [inline NetKAN/Arkas.netkan]: `"Arkas"` needs to be `"GameData/Arkas"`. The folder structure is `Arkas/GameData/Arkas` so CKAN will `find` the top-level `Arkas` folder and install everything inside it.

File: NetKAN/Arkas.netkan [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	"license"  	:	"CC-BY-NC-SA",
+	"comment"	:	"PlanetPack",
+	"abstract"	:	"This Mod adds a single planet to the Kerbol system between the orbits of Moho and Eve.  A thick atmosphere on a desert world with 17 unique biomes - Optional Cloud Support",
+	"resources" :
+	{
+		"homepage"	:	"http://forum.kerbalspaceprogram.com/threads/134547",
+		"kerbalstuff"	:	"https://kerbalstuff.com/mod/1158/Arkas%20-%20A%20Thick%20Atmosphere%20on%20a%20Desert%20World"
+	},
+	"depends"	:	[ { "name"	:	"Kopernicus", "min version": "0.4" } ],
+	"recommends"	:	[ { "name"	:	"Arkas-Config" } ],
```
Comment (plague006) [inline NetKAN/Arkas.netkan]: `"Arkas-Config"` should be `"Arkas-Clouds"` to be more descriptive.

File: NetKAN/Arkas.netkan [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+	"identifier"	:	"Arkas",
+	"$kref"		:	"#/ckan/kerbalstuff/1158",
+	"spec_version"	:	"v1.4",
+	"license"  	:	"CC-BY-NC-SA",
+	"comment"	:	"PlanetPack",
+	"abstract"	:	"This Mod adds a single planet to the Kerbol system between the orbits of Moho and Eve.  A thick atmosphere on a desert world with 17 unique biomes - Optional Cloud Support",
+	"resources" :
```
Comment (plague006) [inline NetKAN/Arkas.netkan]: The entire `resources` section should be removed, this information will be scraped from KerbalStuff.

File: NetKAN/Arkas.netkan [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+	"identifier"	:	"Arkas",
+	"$kref"		:	"#/ckan/kerbalstuff/1158",
+	"spec_version"	:	"v1.4",
+	"license"  	:	"CC-BY-NC-SA",
```
Comment (plague006) [inline NetKAN/Arkas.netkan]: The license is `CC-BY-NC-SA-4.0` not `CC-BY-NC-SA`

File: NetKAN/Arkas-Config-Med-Clouds.netkan [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{
+	"spec_version"	:	1,
+	"identifier"	:	"Arkas-Config-Med-Clouds",
+	"name"		:	"Arkas Med Cloud configuration",
+	"$kref"		:	"#/ckan/kerbalstuff/1158",
+	"license"  	:	"CC-BY-NC-SA",
```
Comment (plague006) [inline NetKAN/Arkas-Config-Med-Clouds.netkan]: The clouds are not licensed under `CC-BY-NC-SA`, the license is `unknown` (see LICENSE_Clouds.txt).

File: NetKAN/Arkas-Config-Med-Clouds.netkan [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+	"abstract"	:	"Optional cloud support.  This includes the medium level cloud cover",
```
Comment (plague006) [inline NetKAN/Arkas-Config-Med-Clouds.netkan]: Abstract should read "Medium clouds for Arkas" to be more descriptive.

File: NetKAN/Arkas-Config-Med-Clouds.netkan [hunk 7]
```diff
@@ -0,0 +1,21 @@
+{
+	"spec_version"	:	1,
+	"identifier"	:	"Arkas-Config-Med-Clouds",
+	"name"		:	"Arkas Med Cloud configuration",
```
Comment (plague006) [inline NetKAN/Arkas-Config-Med-Clouds.netkan]: name does not need the word "configuration". `"Arkas - Medium Clouds"` would be sufficient.

File: NetKAN/Arkas-Config-Med-Clouds.netkan [hunk 8]
```diff
@@ -0,0 +1,21 @@
+{
+	"spec_version"	:	1,
+	"identifier"	:	"Arkas-Config-Med-Clouds",
```
Comment (plague006) [inline NetKAN/Arkas-Config-Med-Clouds.netkan]: `Arkas-Clouds-Medium` would be a more descriptive ID and help with location and sorting of the metadata files.

File: NetKAN/Arkas-Config-Med-Clouds.netkan [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+		{
+			"file"		:	"Arkas/Cloud Support/Med Cloud Cover Config/GameData/BoulderCo",
+			"install_to"	:	"GameData"
+		},
+		{
+			"file"		:	"Arkas/Cloud Support/GameData/BoulderCo",
+			"install_to"	:	"GameData"
+		}
+	],
+	"depends"	:	[ { "name"	:	"Arkas" } ],
```
Comment (plague006) [inline NetKAN/Arkas-Config-Med-Clouds.netkan]: I might be wrong and someone can correct me if I am, but I believe all clouds depend on `EnvironmentalVisualEnhancements` unless they write their own .dll which Arkas does not.

**Comment by plague006** [comment]
@Olympic1 if you want to double check the relationship with other Kopernicus bodies, I'll make sure the metadata is acceptable before merge.

File: NetKAN/Arkas.netkan [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+	"comment"	:	"PlanetPack",
+	"abstract"	:	"This Mod adds a single planet to the Kerbol system between the orbits of Moho and Eve.  A thick atmosphere on a desert world with 17 unique biomes - Optional Cloud Support",
```
Comment (plague006) [inline NetKAN/Arkas.netkan]: `abstract` will be scraped from KerbalStuff and should not be hardcoded. Whenever possible and practical we allow authors to describe their own mods.

**Comment by Olympic1** [comment]
@plague006 Arkas is compatible with all planet packs. As for the dependencies, it does require `EnvironmentalVisualEnhancements` to work.

File: NetKAN/Arkas-Config-Med-Clouds.netkan (hunk 8 above)
Comment (linuxgurugamer) [inline NetKAN/Arkas-Config-Med-Clouds.netkan]: And here I thought I had done such a good job.

OK, I've implemented all the changes in all 4 files, and renamed the 3 
config files to match the new identifier

Thanks


JBB

File: NetKAN/Arkas.netkan (hunk 2 above)
Comment (Olympic1) [inline NetKAN/Arkas.netkan]: @linuxgurugamer, You forgot to change this line. Change to `Arkas-Clouds`

File: NetKAN/Arkas.netkan [hunk 11]
```diff
@@ -0,0 +1,24 @@
+{
+	"identifier"	:	"Arkas",
+	"$kref"		:	"#/ckan/kerbalstuff/1158",
+	"spec_version"	:	"v1.4",
+	"license"  	:	"CC-BY-NC-SA-4.0",
+	"comment"	:	"PlanetPack",
+	"resources" :
```
Comment (Olympic1) [inline NetKAN/Arkas.netkan]: Also forgot to remove the `resources` section

Comment (linuxgurugamer) [inline NetKAN/Arkas.netkan]: hmmm.  Sorry, I was rushing to a dr. appt.

Ok, changes made

Thanks

and, in general, a big thanks for all the work you do for this project.


JBB

**Comment by Olympic1** [comment]
Merging with great thanks

**Comment by linuxgurugamer** [comment]
ThankYOU!


JBB

**Pull request merged**